book/tables/pharmacokinetic/pkct01.qmd

```diff
@@ -130,7 +128,6 @@ lyt <- lyt_rows %>%
       n = "n", n_blq = "Number\nof\nLTRs/BLQs", mean = "Mean", sd = "SD", cv = "CV (%) Mean",
       geom_mean = "Geometric Mean", geom_cv = "CV % Geometric Mean", median = "Median", min = "Minimum", max = "Maximum"
     ),
-    cache = TRUE,
```

Comment:
Do you think it is better not to have this?

Reply:
It makes the TLG-C example rendering more efficient. BUT if someone copies the code and uses it on a project they may get the wrong results. So I think it's safer to remove it from user-facing docs.

Reply:
It makes perfect sense! Thanks Jana ;)